src/applications/travel-pay/components/SmocRadio.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { VaRadio } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+
+export default function SmocRadio({
```

Comment:
Should we add tests for this component?

Reply:
I considered it, but it already had full coverage from the normal SMOC flow tests so I kept it concise here. I'm down to add some just for completeness sake if you'd like.

Reply:
Huh, good point. This is a reused component within smoc, but won't probably be used outside smoc, so we're probably good if it's fully covered by other smoc tests.